justfile

```diff
@@ -4,7 +4,7 @@ install:
 
 # run unit tests
 test:
-    poetry run pytest
+    poetry run pytest --cov=nominal --cov-branch --cov-report=html --cov-report=term --cov-config=.coveragerc
```

Comment:
is this config possible in pyproject.toml or nah?

Reply:
yes it is! i was gonna do a bigger pyproject setup PR, but can add this in here